### PR TITLE
Added a new license.

### PR DESCRIPTION
**Pull request by LaurenceOfAustralia**
Created Modified Autonomous Software License (MASL).

### CODE REVIEW

**Comment by viktorsec** [comment]
Planned to add a modified version of ASL in the evening, nice work! 👏

File: LICENSE.txt [hunk 1]
```diff
@@ -1,16 +1,59 @@
...
+conditions are imposed on you (whether by court order, agreement or otherwise) that contradict the conditions of this License, they do not
+excuse you from the conditions of this License.  If you cannot distribute so as to satisfy simultaneously your obligations under this
+License and any other pertinent obligations, then as a consequence you may not distribute or edit the Autonomous Software at all.
+
+6. You are not required to accept this License, since you have not signed it.  However, nothing else grants you permission to modify or
+distribute the Autonomous Software or its derivative works.  These actions are prohibited by law if you do not accept this License.
+Therefore, by modifying or distributing the Autonomous Software (or any work based on the Autonomous Software), you indicate your acceptance of this License to do so, and
+all its terms and conditions for copying, distributing or modifying the Autonomous Software or works based on it.
+
+5. The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Autonomous Software.
```
Comment (viktorsec) [inline LICENSE.txt:54]: Should be 7.

File: LICENSE.txt [hunk 2]
```diff
@@ -1,16 +1,59 @@
...
+2. Grant of Copyright License. Subject to the terms and conditions of this License, the Autonomous Software hereby grants to You a perpetual,
+worldwide, non-exclusive, no-charge, royalty-free, irrevocable copyright license to reproduce, prepare Derivative Works of,
+publicly display, publicly perform, sublicense, and distribute the Work and such Derivative Works in Source or Object form.
+
+3. Redistribution. You may reproduce and distribute copies of the Work or Derivative Works thereof in any medium, with or without
+modifications, and in Source or Object form, provided that You meet the following conditions:
+
+      (a) You must give any other recipients of the Work or Derivative Works a copy of this License; and
+
+      (b) You must cause any modified files to carry prominent notices stating that You changed the files; and
```
Comment (viktorsec) [inline LICENSE.txt:30]: Why?

**Review by viktorsec — COMMENTED** [review]
I don't like the restrictions in point (3). I think that the user should be able to do whatever they want.

File: LICENSE.txt [hunk 3]
```diff
@@ -1,16 +1,59 @@
-MIT License
+Modified Autonomous Software License (MASL)
```
Comment (jblz) [inline LICENSE.txt:1]: y u no GPL?

**Review by jblz — CHANGES_REQUESTED** [review]

File: LICENSE.txt (hunk 3 above)
Comment (viktorsec) [reply LICENSE.txt:1]: Because GPL is too restrictive.

https://github.com/chaosbot/chaos/issues/220#issuecomment-303960463

Comment (greenboxal) [reply LICENSE.txt:1]: Because GPL is like cancer: it spreads everywhere

File: LICENSE.txt (hunk 2 above)
Comment (EtherTyper) [reply LICENSE.txt:30]: :-1: The original author made it MIT licensed, without any Apache-like modification notice requirement like the "MASL" has. Also, you need to attribute the original author somewhere.

File: LICENSE.txt (hunk 3 above)
Comment (flibustier) [reply LICENSE.txt:1]: What about « Common Heroic Autonomous Open Software » ? 😄

**Comment by bengjerstad** [comment]
👎 Funny, but Number 3 is too restrictive.

**Comment by LaurenceOfAustralia** [comment]
I may make a modified version getting rid of number 3

**Pull request closed without merging**